Dockerfile

```diff
@@ -26,6 +26,10 @@ RUN npm config set cafile "/app/DOIRootCA2.cer" \
 
 RUN node_modules/bower/bin/bower install --allow-root
 
+RUN nosetests --all-modules --exe
+
+RUN ls *
```

Comment:
What is the purpose of this? I suspect it's for debugging and should probably be removed.